static/app/components/searchSyntax/grammar.pegjs

```diff
@@ -293,24 +293,39 @@ text_key
 
 value
   = value:[^()\t\n ]* {
-      return tc.tokenValueText(value.join(''), false);
+      return tc.tokenValueText(value.join(''), false, false);
     }
 
 quoted_value
-  = '"' value:('\\"' / [^"])* '"' {
-      return tc.tokenValueText(value.join(''), true);
+  = '"' value:('\\"' / '\\*' / [^"\\])* '"' {
+      return tc.tokenValueText(value.join(''), true, false);
     }
 
+contains_value
+  = "*" value:(quoted_value / contains_inner_value)* "*" {
```

Comment:
This looks like it won't catch a quoted contains value? I'd suggest adding a test for a filter that has a contains value that has a space in it (which would necessitate wrapping quotes). Something like `foo:"*bar baz*"`

Reply:
Okay, re-did how quoted values are handled, so now they are their own thing and should be handled better.